content/working_with/console/pages/local-blueprints-page.md

```diff
@@ -1,22 +1,33 @@
 ---
 layout: bt_wiki
-title: Local Blueprints Page
+title: Blueprints Page
 category: Console
 draft: false
 abstract: Blueprint Page Reference
 weight: 130
 aliases: ["/manager_webui/blueprints-page/", "/working_with/console/blueprints-page/", "/working_with/console/local-blueprints-page/"]
 ---
 
-The Local Blueprints page acts as the catalog of services (blueprints) that are available for use on this {{< param cfy_console_name >}}. It provides a list of the blueprints uploaded to the {{< param cfy_manager_name >}} and action buttons to upload, deploy and delete them. It also allows to open [{{< param cfy_composer_name >}}]({{< param cfy_composer_link >}}) or edit specific blueprint in [{{< param cfy_composer_name >}}]({{< param cfy_composer_link >}}).
+The Blueprints page acts as the catalog of services (blueprints) that are available for use on this {{< param cfy_console_name >}}.
+It provides a list of the blueprints uploaded to the {{< param cfy_manager_name >}} and action buttons to upload, deploy and delete them.
+It also allows to edit specific blueprint in [{{< param cfy_composer_name >}}]({{< param cfy_composer_link >}}).
 
 {{% note %}}
-[{{< param cfy_composer_name >}}]({{< param cfy_composer_link >}}) is only available in [{{< param mgr_premium_title >}}]({{< param mgr_premium_link >}}).
+The "edit in [{{< param cfy_composer_name >}}]({{< param cfy_composer_link >}})" option is only available in [{{< param mgr_premium_title >}}]({{< param mgr_premium_link >}}).
 {{% /note %}}
```

Comment:
Actually right now it is turned off by default (as **Show Composer options** is set to Off in Blueprints widget configuration), so maybe we should not add such information?

Reply:
Agree, removed.